includes/class-wc-stripe-helper.php

```
@@ -1116,8 +1116,45 @@ public static function has_cart_or_checkout_on_current_page() {
 	 * @return bool True if the account is not in supported countries.
 	 */
 	public static function is_adaptive_pricing_available_for_account(): bool {
-		$account_country = WC_Stripe::get_instance()->account->get_account_country();
-		return strtoupper( $account_country ) !== WC_Stripe_Country_Code::INDIA;
+		$reason = self::get_adaptive_pricing_account_availability_reason();
+		if ( null === $reason ) {
+			return true;
+		}
+
+		return false;
+	}
```

PHPMD (2.15.0)

[error] The method is_adaptive_pricing_available_for_account is not named in camelCase. (undefined)

(CamelCaseMethodName)